encrypted-media-respec.html

```diff
@@ -1937,9 +1940,16 @@
                           <p>
                             Destroy the license(s) and/or key(s) associated with the session.
                           </p>
+                          <p class="note">
+                            This implies destruction of the license(s) and/or keys(s) whether they are in memory or in persistent store or both.
```

Comment:
nit: The second "in" is probably unnecessary.

This might be better as "...whether they are in memory, persistent store, or both."